Mail/Views/Bottom sheets/Discovery/DiscoveryView.swift

```diff
@@ -74,16 +82,29 @@ struct DiscoveryView: View {
     @State private var willDiscoverNewFeature = false
 
     let item: DiscoveryItem
+    var isShowingLaterButton = true
 
     var onAppear: (() -> Void)?
     let completionHandler: (Bool) -> Void
 
+    private var laterButtonAction: (() -> Void)? {
```

Comment:
I am not a big fan of this strategy. Maybe me should think this view differently
Computing an optional closure to show/hide a button seems a bit odd